en/development/testing.rst

```diff
@@ -312,6 +312,14 @@ This will put the coverage results in your application's webroot directory. You
 should be able to view the results by going to
 ``http://localhost/your_app/coverage``.
 
+If you are using PHP 5.6.0 or greater, you can use `phpdbg <http://phpunit.de/manual/current/en/installation.html#installation.phar.windows>`__ 
+to generate coverage instead of xdebug. ``phpdbg`` is generally faster at 
```

Comment:
Whether it's relevant or not is totally up for debate. I hesitate to put it here as it's an additional tool and isn't cakephp related. However, there is other non-cakephp information (like this section in the first place) so I thought it might be helpful.